src/client/views/play.ejs

```diff
@@ -494,6 +495,13 @@
                         <div id="guardsB" class="promotepiececontainer hidden"><div class="promotepiece guardsB button"></div></div>
                     </div>
                 </div>
+                
+                <!-- Draw Offer UI -->
+                <div id="draw_offer_ui" class="draw_offer_ui footer hidden">
```

Comment:
The draw offer UI needs to be moved to where my next comment says

Reply:
Why

Reply:
Actually it won't let me make a comment there so this is where it needs to go:

<img width="716" alt="Screen Shot 2024-07-31 at 11 55 26 AM" src="https://github.com/user-attachments/assets/f236078e-1023-4ec3-8127-65594c989528">

Then, whenever the draw UI needs to reveal, in guigameinfo.js you have to use style.hideElement(element_whosturn) to hide the other words.

Reply:
I tried to do that earlier, but it didn't work. Why do we need to move the UI up there? Shouldn't we be able to just hide the text but keep the UI where it is?

Reply:
What didn't work about it? The way you are doing it now, the UI is a separate object, overlayed on top of the already existing bar, it doesn't feel natural, it should be integrated into it, it should be a part of it.

Reply:
We do hide the text, only the draw offer UI or the whosturn element is visible at one time